src/compile-middleware.ts

```diff
@@ -62,7 +62,7 @@ const compileMimeTypes = [
 export const isPolyfill = /(^|\/)webcomponentsjs\/[^\/]+$/;
 
 function getContentType(response: Response) {
-  const contentTypeHeader = response.getHeader('Content-Type');
+  const contentTypeHeader = response.getHeader('Content-Type') as string;
```

Comment:
I fixed these a different way in https://github.com/Polymer/polyserve/pull/210

I converted the getHeader('x') calls to get('x') which had the correct typings info and this wasn't necessary.